tests/unit/Alteration.spec.ts

```diff
@@ -383,6 +383,30 @@ describe('Alteration component', () => {
     expect(store.state.stateModel.currentFees.futureEffectiveFees).toBe(100)
   })
 
+  it('certify text is not prefilled/editable for staff user', async () => {
+    store.state.stateModel.tombstone.keycloakRoles = ['staff']
+    store.state.stateModel.tombstone.userInfo = {
+      firstname: 'Jon',
+      lastname: 'Doe'
+    }
+    await wrapper.setProps({ appReady: true })
+    await flushPromises()
```

Comment:
OK, it does make sense that you need flushPromises so that onAppReady() does everything it needs to (all its promises need to be resolved/flushed).

I think, in some cases, `valid` is initialized to false, not by the sub-component but by the store.

Anyway, I'm OK with this. Resolving.